Backend/UnitTesting/PointServiceUT.cs

```diff
@@ -66,16 +67,16 @@ public void Update_Point_Test_Longtitude_Latitude_Limits()
 
             //testing invalid values for longitude and latitude
             newPoint = tu.CreatePointObject(-181, -90);
-            Assert.ThrowsException<ArgumentOutOfRangeException>(() => ps.UpdatePoint(_db, newPoint));
+            Assert.ThrowsException<InvalidPointException>(() => ps.UpdatePoint(_db, newPoint));
 
             newPoint = tu.CreatePointObject(-180, -91);
-            Assert.ThrowsException<ArgumentOutOfRangeException>(() => ps.UpdatePoint(_db, newPoint));
+            Assert.ThrowsException<InvalidPointException>(() => ps.UpdatePoint(_db, newPoint));
 
             newPoint = tu.CreatePointObject(181, 90);
-            Assert.ThrowsException<ArgumentOutOfRangeException>(() => ps.UpdatePoint(_db, newPoint));
+            Assert.ThrowsException<InvalidPointException>(() => ps.UpdatePoint(_db, newPoint));
 
             newPoint = tu.CreatePointObject(180, 91);
-            Assert.ThrowsException<ArgumentOutOfRangeException>(() => ps.UpdatePoint(_db, newPoint));
+            Assert.ThrowsException<InvalidPointException>(() => ps.UpdatePoint(_db, newPoint));
         }
 
         [TestMethod]
```

Comment:
Updated to reflect change in PointService with specific point error instead of ArgumentOutOfRangeException.